foreman_ansible.gemspec

```diff
@@ -22,4 +22,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'deface', '< 2.0'
   s.add_dependency 'foreman_remote_execution', '>= 9.0', '< 13'
   s.add_dependency 'foreman-tasks', '>= 7.0', '< 10'
+  s.add_dependency 'git', '~> 1.0'
```

Comment:
I'd prefer to avoid the whole git dependency. See more in the overall review.

Reply:
All git functionality has been moved to the smartproxy plugin. I removed the dependency.